src/AssetRegistry/MainRegistry.sol

```diff
@@ -171,6 +176,54 @@ contract MainRegistry is Ownable {
         return true;
     }
 
+    function batchProcessDeposit(
```

Comment:
Function replaces batchIsWhiteListed(), shouldn't we remove the latter?

Reply:
batchIsWhiteListed is a view function, should we keep it for frontends?